### PR TITLE
Bug 1939815: Modified warning Alert for Encrypted PVs in Create StorageClass

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

File: frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx [hunk 1]
```diff
@@ -440,7 +440,7 @@ export const StorageClassEncryption: React.FC<ProvisionerProps> = ({ onParamChan
                 className="co-alert"
                 variant="warning"
                 title={t(
-                  'ceph-storage-plugin~Encrypted PVs cannot be cloned expanded or create snapshots.',
+                  'ceph-storage-plugin~PV expansion, snapshot and cloning operations are not supported for encrypted PVs.',
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx:443]:
```suggestion
                  'ceph-storage-plugin~PV expansion operation is not supported for encrypted PVs.',
```

Comment (GowthamShanmugam) [reply frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx:443]: ack

**Pull request merged**